### PR TITLE
remove vendor directory, update pins

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by bcreane — APPROVED** [review]
very nice! Mostly deleted lines because of the shift to our data-plane-api fork.

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

File: go.mod
Comment (bcreane) [inline go.mod:3]: isn't our go-build image up to go 1.15?

Comment (caseydavenport) [reply go.mod:3]: Yeah, I thought I updated this but I guess I didn't

**Pull request merged**
